Makefile

```diff
@@ -87,6 +87,14 @@ lint: golangci-lint yamllint ## Run golangci-lint linter & yamllint
 lint-fix: golangci-lint ## Run golangci-lint linter and perform fixes
 	$(GOLANGCI_LINT) run --fix
 
+.PHONY: lint-testdata
+lint-testdata: golangci-lint ## Run golangci-lint for the lscaffolded projects under testdata directory
+	./lint-testdata.sh
+
+.PHONY: lint-testdata-fix
+lint-testdata-fix: golangci-lint ## Run golangci-lint for the lscaffolded projects under testdata directory
+	./lint-testdata.sh fix
```

Comment:
I do not think so, we should not run fix-lint in the CI
It is ONLY a helper for us in the develop env and does not fix all issues automatically.
It can only fix issues like "go fmt" and extra imports.

Reply:
>It is a helper for dev env and not be called in the CI.
We should not try to fix issues in the CI at all.
Also, it does not fix all issues automatically.
It can only fix issues like "go fmt" and extra imports.
>
>I mean:
>
  >  In the CI we just check
  >  Locally, in dev env, we can use make lint-fix to help out.
>
>It is for we have the same make lint-fix that we have for the KB code to the samples either.

Yeah, but all the projects under the `testdata/` directory are automatically generated. Using a lint fixing command will fix the problem but the problem will come back when the testdata is regenerated if it isn't fixed in the actual scaffold logic.